app/Config/Routes.php

```diff
@@ -55,6 +55,8 @@
 $routes->post('dashboard/admin/event/new', [AdminDashboard::class, 'createEvent'], ['filter' => AdminAuthFilter::class]);
 
 $routes->get('dashboard/admin/approval/speaker', [Approval::class, 'getPendingSpeakers'], ['filter' => AdminAuthFilter::class]);
+$routes->get('dashboard/admin/approval/user/(:num)/event/(:num)/can-reject', [Approval::class, 'canReject'], ['filter' => AdminAuthFilter::class]);
```

Comment:
This means that I have to make a API call for every speaker entry in the approval page, right?
Or do I miss something?

Reply:
You should be right, yes. If you do all of them async, this shouldn't be a problem (I hope).